src/main/resources/application.yml

```diff
@@ -1,8 +1,8 @@
 spring:
   datasource:
-    url: jdbc:${DATABASE_DRIVER}://${DATABASE_URL}:${DATABASE_PORT}/${DATABASE_NAME}?useSSL=false&serverTimezone=UTC&useLegacyDatetimeCode=false
-    username: ${DATABASE_USERNAME}
-    password: ${DATABASE_PASSWORD}
+    url: jdbc:postgresql://localhost:5432/junior?useSSL=falseserverTimezone=UTC&useLegacyDatetimeCode=false
+    username: postgres
+    password: ZAQ!2wsx
```

Comment:
Tutaj jakbyś podał swoje hasło do bazy takie jakie wykorzystujesz na co dzień to raczej byłoby niefajnie je tu puchnąć :D